### ISSUE TITLE
The directory name is invalid error when launching task defined in package.json

### ISSUE THREAD

**Issue by nealoke** [lkytal/quickTask#18]
Hi, I just updated to the latest version of vscode (and probably this extension) but now when I try to launch a task it gives me the following error: `The directory name is invalid.`

I noticed that the info changed when launching quicktask. Now it includes the filename (in this case `package.json`) in the drop down list. When launching it tries to `cd directory/package.json` which obviously does not work :)

I hope we can quickly work this out as it is a major pain launching these individual tasks

Looking at the source code, I guess something goes wrong in retrieving the `targetTask.relativePath` in **[extension.ts](https://github.com/lkytal/quickTask/blob/master/src/extension.ts#L62)**

```
if (targetTask.relativePath != null && targetTask.relativePath != "") {
	terminal.sendText('cd ' + targetTask.relativePath);
}
```

**Comment by lkytal** [comment]
Should be fixed in version 3.1.1

**Comment by nealoke** [comment]
@lkytal Thank you! Btw, this plugin is such a productivity booster and thanks for putting in the time to make this!
